### DISCUSSION TITLE
Addition of Wheat Beer Brewing

### DISCUSSION THREAD

**Discussion by jasper-one** [EveronLife/EveronLife#104] [Ideas]
### Summarize your proposed improvement

I'm proposing the addition of another Profession that would leverage existing assets to create Beer Brewing. This would give us another legal profession for the mid-game. 

**Gathering**
The gathering could be done from specific fields or new fields implemented on the map:
![image](https://user-images.githubusercontent.com/106525617/172306615-81c20f00-fb30-4d6f-82e2-4749a0ba6de4.png)

**Processing**
Processing would require Empty Beer Bottles and Wheat. The processor would be the existing Beer Brewing Kettles at a "Brewery" location:
![image](https://user-images.githubusercontent.com/106525617/172306376-e839d810-891a-4b23-916f-e8179c268c0a.png)

**Selling**
Conceptual only until #8 is added but beers could be sold directly at locations such as restaurants (and of course wherever server owners want).
![image](https://user-images.githubusercontent.com/106525617/172307450-1d5373ce-f335-4c0c-9957-e9f42297638e.png)

I'm happy to implement this and if #101 is merged I can make this require a Shovel if @12gabriel3 or another modeler are keen on adding another tool to the game. 

### Why is this needed?

Generally speaking, I'm looking forward to providing the community with many different professions they can use out of the box. 

### Describe alternatives you've considered

Standard Wheat gathering and selling as Bales of Hay could also be included alongside using the hay bales for brewing beers.
![image](https://user-images.githubusercontent.com/106525617/172306512-26a04c10-5c2a-42db-9b76-689d5c755c6b.png)

**Comment by Arkensor** [comment]
As a general side note, features that require no additional assets are very welcome. But keep in mind that we do not need 30 different jobs in the core framework. server owners can do "similar" jobs with that ever assets they need. So for the core framework, it would be nice to have 2-3 jobs of each "activity type" as nice default examples.
So I am not sure if something like 5 is not too much. Depends on how awesome they are. maybe we should only pick the most favorite/diverse 3 of them
We can have horizontal progression later, where all the systems are worked out to include more variety by default.

**Comment by Arkensor** [comment]
With so many base items already available, I think this will definitely be on the list of "included with the framework".
While we are waiting for the processing and trader PR's - how do people get empty bottles? I think we could use this for making multiple jobs at once. 
- One is to collect bottles from vendors around the map and maybe garbage cans? You bring them all to the brewery and get paid. Alternatively, people could also buy new bottles from a factory that makes them (Sand -> Bottles)? If you have time collecting them saves you some money, but if you want to quickly make cash you maybe get new ones?
- One is to gather the wheat and sell it to the brewery (as one option besides a bakery that produces flour or something)
- For when we have advanced farming people could set up an improvised brewery in their backyard etc to brew the beer themselves and sell that directly to vendors?
- One is to buy beer from the brewery directly, drive to vendors and sell it for a small markup maybe.
The "static" brewery processes a much larger volume of things you deliver, meanwhile backyard operations might take longer and become only a small side activity you can do?

This would demonstrate the concept of shared resources (wheat being useable for multiple products: beer vs bread), an interesting way to gather a base resource (collecting bottles around the map and the ability to buy them to save time but get less money) and using a factory for processing vs a player-owned setup.

These are just some thoughts I had. Of course, people can just process wheat into bottles with beer in it as output, but I see lots of potential here. But if done like this, it would be part of a very late advanced farming + player housing / owned businesses update? We could do the basic version at first of course.

**Reply by jasper-one** [reply]
Yeah, I really like these ideas @Arkensor. I might try and flesh them out more here and make this more of an advanced idea that we can work to while adding some more basic chains under different idea/discussion posts.

**Comment by 12gabriel3** [comment]
The game already comes with shovel models, we could make an item using them.

![image](https://user-images.githubusercontent.com/16885097/173676859-ccc1ced2-9a7c-42bb-acd3-cdfb613d4f18.png)

**Reply by jasper-one** [reply]
That's awesome, we should 100% leverage that. I like the small one.

**Reply by jasper-one** [reply]
What is that one on the right called @12gabriel3 ?

**Reply by Arkensor** [reply]
Very good find @12gabriel3. We will have to see if we can support two-handed shoveling animation AND one-handed, but I think that could be doable. Then we can use thee built-in assets, they are timeless and look very detailed.
